tsconfig.json

```diff
@@ -20,7 +20,6 @@
     // "sourceMap": true,                     /* Generates corresponding '.map' file. */
     // "outFile": "./",                       /* Concatenate and emit output to single file. */
      "outDir": "./build",                        /* Redirect output structure to the directory. */
-     "rootDir": "./src",                       /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
```

Comment:
for deployment we want only the `src` directly to be compiled, else it will include the tests file too.
does it make any problems if we leave this `rootDir` key?